conf.yaml

```
@@ -996,10 +996,10 @@ contents:
                 path:   docs/asciidoc
           - title:      Elastic Common Schema (ECS) Reference
             prefix:     en/ecs
-            # Please do not update current to 8.12
+            # Please do not update current to 8.17
             current:    8.11
```

We can bump this now as 8.16 has been released

```suggestion
            current:    8.16
```